bedrock/firefox/templates/firefox/dev-whatsnew.html

```diff
@@ -68,49 +81,18 @@
           <a href="https://hacks.mozilla.org/?p=28808" rel="external" class="more">{{_('See how you helped')}}</a>
```

Comment:
As discussed with @flodolo on IRC - we're still going to merge this as-is and push all strings out for l10n.